tests/test_changes.list

```diff
@@ -1,5 +1,19 @@
-hafs_regional_atm_ocn intel
-hafs_regional_docn intel
-hafs_regional_docn_oisst intel
-control_p8_atmlnd intel
-control_restart_p8_atmlnd intel
+control_CubedSphereGrid intel
+control_CubedSphereGrid_parallel intel
+control_latlon intel
+control_wrtGauss_netcdf_parallel intel
+control_c192 intel
+control_c384 intel
+merra2_thompson intel
+control_CubedSphere_debug intel
+control_diag_debug intel
+atmaero_control_p8_rad_micro intel
+control_c48 gnu
+control_debug_p8 gnu
+control_diag_debug gnu
+control_p8 gnu
+control_p8_ugwpvl gnu
+cpld_control_nowave_noaero_p8 gnu
```

Comment:
In your original spreasheet, aren't these listed as passing? What changed?

Reply:
I thought I had left a comment explaining this, sorry.  These tests were done on Hera before the full switch over to Rocky8.  It was odd that some of the baselines checked out with the intel compliers but not the gnu compilers. I wasn't able to figure out what was going on before the OS switch on Hera.  I reran the gnu cases on Hercules and the control_p8 cases passed (modified test_changes.list to reflect that).  And I guess we're stuck with trying to rerun the cpld gnu cases until the Rocky8 issues are resolved?

Let me know if you have any suggestions for any next steps.

Reply:
I see, thanks. Since none of your changes involved compilation, and we have equivalent tests on the intel side, U think this is OK.